src/Tvheadend.cpp

```diff
@@ -665,14 +706,20 @@ PVR_ERROR CTvheadend::GetTimerTypes ( PVR_TIMER_TYPE types[], int *size )
 
   /* PVR_Timer.iPreventDuplicateEpisodes values and presentation.*/
   static std::vector< std::pair<int, std::string> > deDupValues;
-  if (deDupValues.size() == 0)
```

Comment:
IMO it's better to be safe than sorry, we cannot rely on Kodi doing the "right thing". Is there a way to change this to non-static so this thing wouldn't be an issue?